lib/checks/aria/aria-required-children.json

```
@@ -21,7 +21,8 @@
       "pass": "Required ARIA children are present",
       "fail": {
         "singular": "Required ARIA child role not present: ${data.values}",
-        "plural": "Required ARIA children role not present: ${data.values}"
+        "plural": "Required ARIA children role not present: ${data.values}",
+        "unallowed": "Element has children which are not allowed"
```

I wonder if we should maybe do something like this:
```suggestion
        "unallowed": "Element has children which are not allowed (see related nodes)"
```

---

This would be a first I believe to point to the related nodes. Is that a pattern we want to adopt?